### PR TITLE
Update bubblesort.c

### PR DESCRIPTION
**Pull request by lumendesp**
I implemented the option to choose the length and elements of the array and select the type of sorting.

### CODE REVIEW

**Comment by Thuva4** [comment]
Please add the tests, and reduce the duplication.

**Comment by lumendesp** [comment]
Sorry, I didn’t understand.

File: algorithms/C/BubbleSort/bubblesort.c [hunk 1]
```diff
@@ -1,38 +1,71 @@
...
+  if(order == 1){
+    for(int i=0; i < size-1; i++){
+      int flag = 0;
+
+      for(int j=0; j<size-1-i; j++){
+        if(array[j] > array[j+1]){
+          aux = array[j]; 
+          array[j] = array[j+1]; 
+          array[j+1] = aux; 
+          flag = 1;
```
Comment (Thuva4) [inline algorithms/C/BubbleSort/bubblesort.c:15]: This is exactly the same as line 31-34, move it to a util function

Comment (lumendesp) [reply algorithms/C/BubbleSort/bubblesort.c]: Done! Check pls :)

**Pull request merged**